edxval/models.py

```diff
@@ -605,6 +605,55 @@ def __unicode__(self):
         return u'{course_id} - {provider}'.format(course_id=self.course_id, provider=self.provider)
 
 
+class OrganizationTranscriptCredentialsState(models.Model):
+    """
+    State of transcript credentials for a course organization
+    """
+    class Meta:
+        unique_together = ('org', 'provider')
+
+    org = models.CharField(verbose_name='Course Organization', max_length=32)
+    provider = models.CharField(
```

Comment:
@muhammad-ammar we don't need to verify provider on pre-save signal. Model will automatically raises Exception if the you try to save it with a Provider which is not in its choices.

Reply:
This is not happening. Model allows to save the incorrect data if I remove the signal.

Reply:
@muhammad-ammar that's may be because of sqlite db.